tests/test_bar/path/bar-aint-recursive

```diff
@@ -0,0 +1 @@
+lemonbar
```

Comment:
It is better to place `ln -s` call in `run_bar_tests.sh`, and put file name into a variable.